osutil/common_test.go

```diff
@@ -0,0 +1,28 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package osutil_test
+
+import (
+	"github.com/snapcore/snapd/osutil"
+)
+
+var truePath = osutil.LookupPathWithDefault("true", "/bin/true")
+var falsePath = osutil.LookupPathWithDefault("false", "/bin/false")
+var gccPath = osutil.LookupPathWithDefault("gcc", "/usr/bin/gcc")
```

Comment:
These should be close to where they are used instead of in a file by themselves.  If they are used in more than one file, it's okay to pick the one which uses them the most, or the one you like the most. :)